src/env/unset.c

```diff
@@ -0,0 +1,37 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*   tosh - 21 Shell                                      ::::::::            */
+/*                                                      :+:    :+:            */
+/*   By: aholster <aholster@student.codam.nl>          +:+                    */
+/*       ivan-tey <ivan-tey@student.codam.nl>         +#+                     */
+/*       nloomans <nloomans@student.codam.nl>        +#+                      */
+/*                                                 #+#    #+#                 */
+/*   License: GPLv3                                ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "private.h"
+
+void					env_unset(t_env *env, char const *const key)
```

Comment:
missing const ptr for env

Reply:
`env_unset(t_env *const env, char const *const key)`